src/Molecules/Header/Header.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useRef } from 'react'
+import { Link as RouterLink } from 'react-router-dom'
+import { Link, Typography, CircularProgress } from '@material-ui/core'
+import { Edit, Save } from '@styled-icons/material'
+
+import Input from '../../Atoms/Input'
+import { useTranslation } from '../../Atoms/Trans'
+import FlexBox from '../../Templates/FlexBox'
+import BottomTooltipIcon from './BottomTooltipIcon'
+
+const Header = ({
+  children,
+  feature = '',
+  featurePath = null,
+  subFeature = '',
+  onChange = () => {},
+  onSave = false,
+  status = null,
+  actions = null,
+  ...props
+}) => {
+  const saveButton = useRef(null)
+  const [isEditing, setEditing] = useState(false)
+  const [isSaving, setSaving] = useState(false)
+  const [value, setValue] = useState(subFeature)
+
+  const t = useTranslation()
+  const displayMode = !isEditing && !isSaving
+  const editMode = isEditing && !isSaving
+
+  const handleValue = value => setValue(value) || onChange(value)
+  const handleCancel = () => handleValue(subFeature) || setEditing(false) || setSaving(false)
```

Comment:
Are we okay with using ` || ` to shorten functions? Because in that case, I do not think they return anything meaningful
Now that I think about it, if onChange returns false, it would break it, right? Is it intended?

Reply:
Indeed, all of these functions returns a `falsy` value, hence the need to combine them using either `||` or `,`.
I wouldn't mind adding a few LF if you think it's better.